mongoose/RestaurantSchema.ts

```diff
@@ -20,7 +20,7 @@ import Restaurant from "../models/Restaurant";
  */
 const RestaurantSchema = new mongoose.Schema<Restaurant>({
     name: {type: String, required: true},
-    ownedBy: {type: mongoose.Schema.Types.ObjectId, ref: "UserModel"},
+    ownedBy: {type: mongoose.Schema.Types.ObjectId, ref: "UserModel", required: true},
     bannerPicture: String,
```

Comment:
Also mentioned in Yutong's pull request - default pics for banner and profile picture?

Reply:
I added default picture in my branch